### PR TITLE
chore: Migrate /localization to camelCase

### PR DESCRIPTION
**Pull request by lipis**
Let the machines do the job

- [x] Delete `replace.txt`, `replace.py`
- [x] Sync with Crowdin

### CODE REVIEW

**Comment by lipis** [comment]
[Green 🍏](https://travis-ci.org/wireapp/wire-webapp/builds/344890543?utm_source=github_status&utm_medium=notification)

File: app/script/entity/message/DecryptErrorMessage.js [hunk 1]
```diff
@@ -37,8 +37,8 @@ z.entity.DecryptErrorMessage = class DecryptErrorMessage extends z.entity.Messag
     this.caption = ko.pureComputed(() => {
       const content = `<span class='label-bold-xs'>${z.util.escape_html(this.user().first_name())}</span>`;
       const string_id = this.is_remote_identity_changed()
-        ? z.string.conversation_unable_to_decrypt_2
-        : z.string.conversation_unable_to_decrypt_1;
+        ? z.string.conversationUnableToDecrypt_2
+        : z.string.conversationUnableToDecrypt_1;
```
Comment (gregor) [inline app/script/entity/message/DecryptErrorMessage.js]: Why the underscore?

File: app/page/template/content/preferences-av.htm [hunk 2]
```diff
@@ -1,14 +1,14 @@
 <div id="preferences-av" class="preferences-page preferences-av" data-bind="with: $root.preferencesAV">
-  <div class="preferences-titlebar" data-bind="l10n_text: z.string.preferences_av"></div>
+  <div class="preferences-titlebar" data-bind="l10n_text: z.string.preferencesAv"></div>
```
Comment (gregor) [inline app/page/template/content/preferences-av.htm]: Let's make it `preferencesAV` in this and all the following cases.

File: app/page/template/modals.htm [hunk 3]
```diff
@@ -305,33 +305,33 @@
...
       <div class="modal-header">
         <div class="modal-close icon-close icon-button pull-right"></div>
-        <div class="modal-title" data-bind="l10n_text: z.string.modal_session_reset_headline"></div>
+        <div class="modal-title" data-bind="l10n_text: z.string.modalSessionResetHeadline"></div>
       </div>
       <div class="modal-text">
-        <span data-bind="l10n_text: z.string.modal_session_reset_message_1"></span>
-        <a data-bind="l10n_href: {support: z.config.SUPPORT.FORM.BUG}, l10n_text: z.string.modal_session_reset_message_link" rel="nofollow noopener noreferrer" target="_blank"></a>
-        <span data-bind="l10n_text: z.string.modal_session_reset_message_2"></span>
+        <span data-bind="l10n_text: z.string.modalSessionResetMessage_1"></span>
```
Comment (gregor) [inline app/page/template/modals.htm]: Why do you want to keep the underscore for numbers?

File: app/script/components/list/conversationListCallingCell.js [hunk 4]
```diff
@@ -59,7 +59,7 @@ z.components.ConversationListCallingCell = class ConversationListCallingCell {
 
     this.users = ko.pureComputed(() => this.conversation.participating_user_ets());
     this.call = ko.pureComputed(() => this.conversation.call());
-    this.call_participants = ko.pureComputed(() => {
+    this.callParticipants = ko.pureComputed(() => {
```
Comment (gregor) [inline app/script/components/list/conversationListCallingCell.js]: Nope

File: app/script/components/list/conversationListCallingCell.js [hunk 5]
```diff
@@ -68,8 +68,8 @@ z.components.ConversationListCallingCell = class ConversationListCallingCell {
     const MAX_DISPLAYED_PARTICIPANTS = 9;
     this.call_participants_rest = ko.observable(0);
     this.call_participants_displayed = ko.pureComputed(() => {
-      const displayed_user_ets = this.call_participants().slice(0, MAX_DISPLAYED_PARTICIPANTS);
-      this.call_participants_rest(this.call_participants().length - displayed_user_ets.length);
+      const displayed_user_ets = this.callParticipants().slice(0, MAX_DISPLAYED_PARTICIPANTS);
+      this.call_participants_rest(this.callParticipants().length - displayed_user_ets.length);
```
Comment (gregor) [inline app/script/components/list/conversationListCallingCell.js]: Nooooo 🙈

File: app/script/components/list/conversationListCallingCell.js [hunk 6]
```diff
@@ -127,7 +127,7 @@ z.components.ConversationListCallingCell = class ConversationListCallingCell {
     };
 
     this.participants_button_label = ko.pureComputed(() => {
-      return z.l10n.text(z.string.call_participants, this.call_participants().length);
+      return z.l10n.text(z.string.callParticipants, this.callParticipants().length);
```
Comment (gregor) [inline app/script/components/list/conversationListCallingCell.js]: No to the second instance in this line

File: app/script/view_model/ModalsViewModel.js [hunk 7]
```diff
@@ -223,7 +223,7 @@ z.viewModel.ModalsViewModel = class ModalsViewModel {
     }
```
Comment (gregor) [inline app/script/view_model/ModalsViewModel.js:223]: L215-217 need to be adjusted as well as we build the string Id on the fly there.

File: app/script/components/list/conversationListCallingCell.js (hunk 4 above)
Comment (lipis) [reply app/script/components/list/conversationListCallingCell.js]: Is that an actual problem?

File: app/script/view_model/ModalsViewModel.js (hunk 7 above)
Comment (lipis) [reply app/script/view_model/ModalsViewModel.js:223]: Fixed

File: app/script/components/list/conversationListCallingCell.js (hunk 4 above)
Comment (lipis) [reply app/script/components/list/conversationListCallingCell.js]: Eventually we will have to rename that

**Pull request merged**